pkg/api/processapi/processapi.go

```diff
@@ -89,6 +89,7 @@ type MsgK8s struct {
 
 type MsgK8sUnix struct {
 	Docker string
+	Cgrpid uint64
```

Comment:
sorry this is opinionated nit: but why not just write `CgroupID` here? Is this user-facing? I honestly wondered why the name by reading it.

Reply:
Because that's what we are already using in MsgK8s: https://github.com/cilium/tetragon/blob/ecdb38ff75056bdef8a7643cd46b3fe4489f9502/pkg/api/processapi/processapi.go#L88